src/components/ClusterDeployment/ClusterDeploymentWizard.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { AlertsContextProvider } from '../AlertsContextProvider';
+import { ClusterDeploymentWizardProps, ClusterDeploymentWizardStepsType } from './types';
+import ClusterDeploymentWizardContext from './ClusterDeploymentWizardContext';
+import ClusterDeploymentDetailsStep from './ClusterDeploymentDetailsStep';
+
+const ClusterDeploymentWizard: React.FC<ClusterDeploymentWizardProps> = ({
+  className = '',
+
```

Comment:
eslint doesnt complain ? Im shocked :)

Reply:
It's a mystery to be solved by a grand master.